sympy/physics/mechanics/body.py

```diff
@@ -191,6 +195,13 @@ def apply_force(self, vec, point=None):
             ...                                   body.frame.y)
             >>> body.apply_force(body.mass * g * body.frame.x, point)
 
+        Notes
+        =====
+
+        If a new force is applied to a body on a point which already has some
+        force applied on it, then the new force is added to the already applied
+        force on that point.
```

Comment:
This isn't how we handle forces/torques in KanesMethod or LagrangesMethod. I think we shouldn't combine forces automatically. The user can do that if needed themselves.

Reply:
I think we should add forces/torques because, like in `test_apply_loads_on_multi_degree_freedom_holonomic_system()` example, the force on 1 Body, is sum of dampner, spring and gravity on all same point. If we don't add them then `loads` would store tuples of same point with different forces.

Reply:
If you sum the forces, then it makes it harder to separate the forces if you need to work with the force list later. There is no reason to sum the forces here. It adds no advantage, only a disadvantage.

> If we don't add them then loads would store tuples of same point with different forces.

Yes, that's what we've done for 10+ years and exactly what KanesMethod and LagrangesMethod expect.

Reply:
In holonomic problem I did, multiple forces are applied and summed together before passing, that's why I added and by this way we can have all forces applied on one point together and by remove_load we can remove the entire load from one point.